src/components/02-components/past-experiences-page/past-experiences-page.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import CardPendingTask from '../card-pending-task/card-pending-task';
+import Link from '../../01-elements/link/link';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+
+type PastExpericiencesPageProps = {
+  experienceInfo: Array<{
+    [index: string]: { experience: string; status: string };
+  }>;
+};
+
+export default function PastExpericiencesPage({
```

Comment:
This is not a full page, only a component within a page. Change the name to `PastExperiencesList` (and the filename accordingly.)